src/javaRosa.js

```diff
@@ -837,6 +838,34 @@ define([
             });
 
         if (options.path === 'labelItext') {
+            if (EXPERIMENTAL_UI) {
+                $input.atwho({
+                    at: "#form",
+                    data: _.chain(mug.form.getMugList())
+                           .map(function(mug) {
+                                return {
+                                    id: mug.ufid,
+                                    name: mug.form.getAbsolutePath(mug, true),
+                                    path: mug.absolutePath
+                                };
+                            })
+                            .filter(function(choice) { return choice.name; })
+                            .value(),
+                    displayTpl: '<li>${name}</li>',
+                    insertTpl: '<output value="${path}" />',
+                    limit: 100,
+                    maxLen: 30,
+                    callbacks: {
+                        matcher: function(flag, subtext) {
```

Comment:
what's this doing? (i did read the docs but don't quite follow them) ...stripping leading whitespace and the "#form" part?

also, nitpicky, but \w includes underscores